src/ComputationContainer/ExceptionHandler/ExceptionHandler.cpp

```diff
@@ -0,0 +1,68 @@
+#include <dlfcn.h>
+
+#include <iostream>
+#include <sstream>
+#include <regex>
+
+#include <boost/stacktrace.hpp>
+
+// qmpc::Log, spdlog cannot be used here,
+// so, this is re-implementing log format.
+
+static std::string getTime()
+{
+    char tt[100];
+    auto t = time(nullptr);
+    tm local;
+    [[maybe_unused]] auto ret =
+        localtime_r(&t, &local);  // ローカル時間(タイムゾーンに合わせた時間)を取得
+    // TODO:osの環境変数にタイムゾーンの設定がない場合は時間がずれる
+    strftime(tt, 256, "%Y-%m-%d %H:%M:%S%z", &local);
+    return std::string(tt);
+}
+
+static std::string generateLogHeader(const std::string file_name, const std::string func_name, const int line)
+{
+    std::stringstream ss;
+    ss << getTime() << " | ERROR | " << file_name << ":" << func_name << ":" << line << " - ";
+    return ss.str();
+}
+
+#define CALL_WITH_DBG_INFO(func) func(__FILE__, __func__, __LINE__)
+
+extern "C"
+{
+
+// to override `throw`, re-define __cxa_throw
+#pragma GCC diagnostic ignored "-Wbuiltin-declaration-mismatch"
+void __cxa_throw(void* ptr, std::type_info* tinfo, void (*dest)(void*))
```

Comment:
This is a pretty dirty hack.
It's best not to do it if you can help it, but if you can't help it, its okay.